hera_mc/cm_hookup.py

```diff
@@ -38,11 +36,17 @@ def __init__(self, session=None):
             self.session = session
         self.handling = cm_handling.Handling(session)
         self.part_type_cache = {}
+        self.hookup_local_file = 'hookup_tmp.npy'
+        self.part_type_cache_file = 'part_type_cache_tmp.npy'
```

Comment:
I do not like these file names since they are relative, so the cache files will land in whatever directory you happen to be running the code in. This means the module will end up dropping little turds all over your working directories over time. You can use the function `os.path.expanduser` to create absolute paths that point to the user's home directory, which will give one cache file per user. I believe that it is safe to assume that the generic cache directory `~/.cache/` exists, so that's probably a good place to store the files.

See also farther down for a way to just use one file instead of two.